common/browser_test.go

```diff
@@ -24,7 +24,7 @@ func TestBrowserNewPageInContext(t *testing.T) {
 	newTestCase := func(id cdp.BrowserContextID) *testCase {
 		ctx, cancel := context.WithCancel(context.Background())
 		logger := log.NewNullLogger()
-		b := newBrowser(ctx, cancel, nil, NewLaunchOptions(false), logger)
+		b := newBrowser(ctx, cancel, nil, NewLaunchOptions(false, false), logger)
```

Comment:
As suggested in a previous comment, I believe making these constants can make it more readable.

Reply:
See [c421358](https://github.com/grafana/xk6-browser/pull/800/commits/c421358a0be874c99dc930ddddefaaa1ee7e12f5).